x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/constants.ts

```diff
@@ -293,52 +314,122 @@ export const SAVE_CUSTOM_VISUAL_WALKTHROUGH_TITLE = i18n.translate(
   }
 );
 
+export const SAVE_CUSTOM_VISUAL_WALKTHROUGH_LINK = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.contentSource.saveCustom.visualWalkthrough.link',
```

Comment:
As per the i18n guidelines, can we change this to:
```suggestion
  'xpack.enterpriseSearch.workplaceSearch.contentSource.saveCustom.visualWalkthrough.linkLabel',
```

Reply:
RE: "id format" replied here: https://github.com/elastic/kibana/pull/112136#issuecomment-919985697